addons/point_of_sale/static/src/app/customer_display_service.js

```diff
@@ -15,11 +15,37 @@ export class LocalDisplay extends Reactive {
         this.globalState = globalState;
     }
     async connect() {
+        if (this.popupWindow && !this.popupWindow.closed) {
+            return;
+        }
+
         this.popupWindow = window.open("", "Customer Display", "height=600,width=900");
-        this.update({ refreshResources: true });
+        if (this.popupWindow && !this.popupWindow.closed) {
+            this.setPopupWindowLastStatus(true);
+            const that = this;
+            this.popupWindow.addEventListener("beforeunload", function (event) {
+                that.setPopupWindowLastStatus(false);
+            });
```

Comment:
```suggestion
            this.popupWindow.addEventListener("beforeunload", () => {
                this.setPopupWindowLastStatus(false);
            });
```

Reply:
Usually the alias used for this is self.

Reply:
Done, thanks !